presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -61,6 +62,7 @@
 
 namespace facebook::presto {
 using namespace facebook::velox;
+using namespace std::chrono;
```

Comment:
Maybe let's not use this namespace to make it a bit more explicit as similar to other part of the codebase.